crates/pipeline-types/src/serde_with_context/serde_config.rs

```diff
@@ -106,16 +126,19 @@ impl From<JsonFlavor> for SqlSerdeConfig {
                 time_format: TimeFormat::Millis,
                 date_format: DateFormat::DaysSinceEpoch,
                 timestamp_format: TimestampFormat::MillisSinceEpoch,
+                decimal_format: DecimalFormat::String,
```

Comment:
everybody is string?

Reply:
in json, yes, we only use the new one for Avro.